### ISSUE TITLE
Replace Instagram Logo with correct black and white and color versions

### ISSUE THREAD

**Issue by mikelax** [grommet/grommet-icons#93]
# Overview

Currently the Grommet icons [Instagram logo](https://icons.grommet.io/?s=instagram) is using an orange color. This should be swapped with the Logo in the [IG Brand Guidelines](https://en.instagram-brand.com/assets)

As a bonus a second logo could be added for the rainbow colored Logo as well. The name could be:  `InstagramColor`.

**Comment by L0ZZI** [comment]
thanks @mikelax for the feedback. I've made the change to the instagram logo so their is a gradient version in the icon library, keep in mind that instagram recommends the black version of the logo since the gradient they use is more of a mesh (a fill type that's a little more complicated) than linear or radial. So I made a change so it will work with a radial.

![instagram](https://user-images.githubusercontent.com/1753301/52496650-c9a39b80-2b88-11e9-90b0-cd89a38b0d53.png)

Just a note, we already support the black version of the logo and really any theme color you want to specify by putting `<Instagram color="black" />`.
